report/topology.go

```diff
@@ -126,10 +147,19 @@ func (t Topology) GetShape() string {
 	return t.Shape
 }
 
+// GetTag returns the current topology tag, or the default if there isn't one.
+func (t Topology) GetTag() string {
+	if t.Tag == "" {
+		return None
```

Comment:
Do we have to set "none" ?  Doesn't it work just as well if you leave it blank?

Reply:
I checked it now and it works fine. I'll change it. Thanks :)